src/router.ts

```diff
@@ -85,6 +101,12 @@ if (domain) {
     { path: '/explore', name: 'explore', component: Timeline },
     { path: '/playground/:name', name: 'playground', component: Playground },
     { path: '/strategy/:name', name: 'strategy', component: Strategy },
+    {
+      path: '/profile/:address',
+      name: 'profile',
+      component: Profile,
+      children: profileRoutes
+    },
```

Comment:
Why no profile routes on custom domains? I saw profile links pointing back to snapshot.org in case of custom domain. Wouldn't it be easier to just add the routes?

Reply:
Profiles have cross space features like, following space, delegators and activity. We could filter activity and hide follow and delegators but this would take away a lot, se we decided to do it like this.